app/controllers/api/v1/reservations_controller.rb

```diff
@@ -0,0 +1,38 @@
+class Api::V1::ReservationsController < ApplicationController
+  before_action :set_api_v1_reservation,only: :destroy
+
+  def index
+    @reservations = current_user.reservations.all
+    render json: @reservations, status: 200
+  end
```

Comment:
- [ ]  **Required!** .It generates the following errors. Kindly fix it. 

![image](https://user-images.githubusercontent.com/65109017/211563048-f33e3e69-a751-4a5c-a2d1-e8d38d455002.png)